src/routes/Acknowledgements.svelte

```diff
@@ -27,8 +27,8 @@
 
 <HeaderWithAnchor id="project-team" level=2>Project Team</HeaderWithAnchor>
 <ul>
-  <li><strong>GSA</strong>: Syed Azeem, Karl Hebenstreit, Joseph Novak, Charles Popelka, Richard Speidel, Sean Zerges.</li>
-  <li><strong>CivicActions</strong>: Owen Barton, Danita Delce, Mike Gifford, Marlena Medford, Daniel Mundra, Farooq Zakhilwal.</li>
+  <li><strong>GSA</strong>: Syed Azeem, Karl Hebenstreit, Andrew Neilson, Joseph Novak, Charles Popelka, Richard Speidel, Sean Zerges.</li>
+  <li><strong>CivicActions</strong>: Owen Barton, Melinda Burgess, Danita Delce, Mike Gifford, Marlena Medford, Daniel Mundra, Jacqueline Quintanilla, Farooq Zakhilwal.</li>
 </ul>
 
 <HeaderWithAnchor id="major-contributors" level=2>Major Contributors</HeaderWithAnchor>
```

Comment:
How is this going to be identified?

Reply:
@mgifford not sure I understand the question.

Reply:
Maybe we should list Betsy, Bruce, under this heading.

Reply:
Right now I think it would be save to add you, me, Owen & Danita as being major contributors to this.  

I think we should either add someone or remove it.  I just don't like blank headings. 

Ultimately it should align with what we have in our CONTRIBUTIONS.txt & AUTHORS.txt. I don't much care how we do this. I'm not sure about adding Betsy here, but possibly NASA.  

Occurred to me that we might also want to add a section around the acknowledgements to do a small colophon where we can highlight the software used to create this site. This would include references to the W3C of course.